Gems/ROS2/Code/Include/ROS2/Frame/ROS2Transform.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/Math/Transform.h>
+#include <AzCore/std/string/string.h>
+#include <geometry_msgs/msg/transform_stamped.hpp>
+
+namespace ROS2
+{
+    //! Publishes transforms as standard ROS2 tf2 messages. Static transforms are published once.
+    //! @note This class is already used through ROS2FrameComponent.
+    // TODO - Rework this class (name, function). Separate broadcaster out of ROS2SystemComponent
```

Comment:
Is this TODO done?
If is not, then it should be made into a GHI: https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#todo-comments